README.md

```diff
@@ -8,244 +8,22 @@ Build ([master](https://github.com/aws/aws-toolkit-vscode/tree/master) branch)|[
 
 The AWS Toolkit for Visual Studio Code is an extension for working with AWS services such as AWS Lambda.
 
-The toolkit is in preview and only available if built from source. This is pre-release software and we recommend against using it in a production environment.
+The toolkit is in developer preview and is available from the [Visual Studio marketplace](https://marketplace.visualstudio.com/itemdetails?itemName=AmazonWebServices.aws-toolkit-vscode). This is pre-release software and we recommend against using it in a production environment.
 
 This is an open source project because we want you to be involved. We love issues, feature requests, code reviews, pull requests or any positive contribution.
 
-## Getting Started
+## User Guide
 
-### Install the AWS SAM CLI
+The [User Guide](https://docs.aws.amazon.com/toolkit-for-vscode/latest/userguide/welcome.html) contains instructions for **installing**, **configuring**, and **using** the toolkit.
```

Comment:
how about "... contains instructions for getting up and running with the Toolkit"   ?
(your call)

Reply:
✔️